contracts/gauges/Gauge.sol

```diff
@@ -81,6 +78,7 @@ contract Gauge is IGauge, ERC2771Context, ReentrancyGuard {
     shares[_user] = _share;
     totalShare = totalShare - _oldShare + _share;
     _updateRewards(_user);
```

Comment:
move `_updateRewards` to line 77

Reply:
_updateRewards only needs to be before updateGainBalance, which only use gainBalance & totalGainBalance

Reply:
moving to L77 will make the code style consistent: updateRewards before any other updates to the user/contract

Reply:
agree.